packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import '../globals.dart';
+import '../runner/flutter_command.dart';
+
+class ConfigCommand extends FlutterCommand {
+  ConfigCommand() {
+    String usageStatus = flutterUsage.enabled ? 'enabled' : 'disabled';
+
+    argParser.addFlag('analytics',
+      negatable: true,
+      help: 'Enable or disable reporting anonymously tool usage statistics and crash reports ($usageStatus).');
```

Comment:
`(currently $usageStatus)` to make it clear whether the parenthetical is the current state or the state that you'll set if you call it